source/guides/developer.rst

```diff
@@ -2,3 +2,49 @@ Mattermost Developer's Guide
 ============================
 
 Detailed developer documentation is available at `https://developers.mattermost.com <https://developers.mattermost.com>`_.
+Detailed information on contributing to the Mattermost open source project. For developer setup documentation `see developers.mattermost.com <https://developers.mattermost.com/contribute/server/developer-setup/>`_.
```

Comment:
Thanks @lieut-data. Are any of the changes to `../developer.rst` intentional?

Reply:
🤦‍♂️ No, I'm just not paying attention here ;P Fixed!